### ISSUE TITLE
Functional tests fail due to SLUGS configuration issue

### ISSUE THREAD

**Issue by PeterHamilton** [OpenKMIP/PyKMIP#480]
The SLUGS-based functional test suite for Travis CI has begun to fail across all branch/PR test runs for specific versions of Python and Ubuntu. These include:

- Python 3.4 on Ubuntu 12.04 LTS
- Python 3.5 on Ubuntu 12.04 LTS / 14.04 LTS / 16.04 LTS
- Python 3.6 on Ubuntu 12.04 LTS / 14.04 LTS / 16.04 LTS
- Python 3.7 on Ubuntu 16.04 LTS

Note that running the functional tests on Python 3.4 on Ubuntu 14.04 LTS / 16.04 LTS appears to work just fine.

The initial error manifested as authentication failures with the PyKMIP server, specifically showing up in the test logs (for example) as:

```
_______ TestSLUGSAuthenticationAndAccessControl.test_policy_live_loading _______
NOTE: Incompatible Exception Representation, displaying natively:

_StringException: Traceback (most recent call last):
  File "/home/travis/build/OpenKMIP/PyKMIP/kmip/tests/functional/services/test_authentication.py", line 117, in test_policy_live_loading
    operation_policy_name="policy_1"
  File "/home/travis/build/OpenKMIP/PyKMIP/kmip/pie/client.py", line 39, in wrapper
    return function(self, *args, **kwargs)
  File "/home/travis/build/OpenKMIP/PyKMIP/kmip/pie/client.py", line 255, in create
    raise exceptions.KmipOperationFailure(status, reason, message)
kmip.pie.exceptions.KmipOperationFailure: OPERATION_FAILED: AUTHENTICATION_NOT_SUCCESSFUL - An error occurred during client authentication. See server logs for more information.
```

The PyKMIP server logs, unfortunately, do not provide enough information to quickly identify the problem:

```
2019-01-18 15:48:20,883 - kmip.server - INFO - Receiving incoming connection from: 127.0.0.1:33409
2019-01-18 15:48:20,884 - kmip.server - INFO - Dedicating session 00000002 to 127.0.0.1:33409
2019-01-18 15:48:20,884 - kmip.server.session.00000002 - INFO - Starting session: 00000002
2019-01-18 15:48:20,885 - kmip.server.session.00000002 - DEBUG - Session cipher selected: ('AES256-SHA', 'TLSv1/SSLv3', 256)
2019-01-18 15:48:20,886 - kmip.server.session.00000002 - DEBUG - Authenticating with plugin: auth:slugs
2019-01-18 15:48:20,894 - kmip.server.session.00000002 - WARNING - Authentication failed.
2019-01-18 15:48:20,895 - kmip.server.session.00000002 - WARNING - Authentication failed.
2019-01-18 15:48:20,897 - kmip.server.session.00000002 - INFO - Stopping session: 00000002
```

For some reason, the server's exception log call leaves no trace in the server log file (see [session.py#L266](https://github.com/OpenKMIP/PyKMIP/blob/master/kmip/services/server/session.py#L266)). Adding in another error log call provides a little more information:

```
2019-01-18 15:48:20,895 - kmip.server.session.00000002 - ERROR - Unrecognized user ID: John Doe
```

A final debugging step using `requests` to hit the top-level SLUGS endpoint reveals that the SLUGS instance set up for the functional tests does not have any user/group data to serve:

```
$ python -c "from __future__ import print_function; import requests; print(requests.get('http://127.0.0.1:8080/slugs/').json())"
{'groups': [], 'users': []}
```

Continue debugging this problem to determine why SLUGS is not being setup correctly. For a reference Travis CI run showcasing the above snippets, see [here](https://travis-ci.org/OpenKMIP/PyKMIP/jobs/481414017).

**Comment by PeterHamilton** [comment]
Temporarily replicating the SLUGS fixes in the PyKMIP Travis CI setup script indicate that it is indeed an upstream SLUGS problem (see [here](https://travis-ci.org/OpenKMIP/PyKMIP/builds/481508159)). Pushing out a new upstream version of SLUGS should fix these issues.

**Comment by PeterHamilton** [comment]
This is fixed as of [SLUGS 1.1](https://github.com/OpenKMIP/SLUGS/tree/release/1.1.0). Upstream fixes resolved these problems.
